pkg/redislocker/new.go

```diff
@@ -0,0 +1,55 @@
+package redislocker
```

Comment:
Some documentation for Godoc would be helpful to have. Both for the package itself and key methods and types.